src/openms/source/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderAlgorithmPicked.cpp

```diff
@@ -1054,7 +1053,7 @@ namespace OpenMS
       {
         s.getFloatDataArrays().erase(s.getFloatDataArrays().begin() + 2);
       }
-      MzMLFile().store("debug/input.mzML", map_);
+      FileHandler().storeExperiment("debug/input.mzML", map_, {FileTypes::MZML});
```

Comment:
allowed types needed here?

Reply:
Not really, but I'm trying to be as consistant as possible